docs/install-upgrade.md

```diff
@@ -20,6 +21,58 @@ Before you install the MongoDB Community Kubernetes Operator, you must:
    ```
    git clone https://github.com/mongodb/mongodb-kubernetes-operator.git
    ```
+4. Review the possible Operator [deployment scopes](#understand-mongodb-community-operator-deployment-scopes) and configure the Operator to watch other namespaces, if necessary.
```

Comment:
Same comment here. "deployment scopes" doesn't seem to link properly.

Reply:
fixed